Mathlib/Analysis/Normed/Ring/WithAbs.lean

```diff
@@ -51,6 +51,26 @@ instance instInhabited : Inhabited (WithAbs v) := ⟨0⟩
 /-- The canonical (semiring) equivalence between `WithAbs v` and `R`. -/
 def equiv : WithAbs v ≃+* R := RingEquiv.refl _
 
+/-- The canonical (semiring) equivalence between `WithAbs v` and `WithAbs w`, for any two
+absolute values `v` and `w` on `R`. -/
+def equivWithAbs (v w : AbsoluteValue R S) : WithAbs v ≃+* WithAbs w :=
```

Comment:
I've called that `WithAbs.equiv₂` in my repo. But maybe `equivWithAbs` is more descriptive (on the other hand, it duplicates the namespace in part of the name.)